modules/ocf_stats/files/prometheus/gen-prometheus-nodes

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+"""Generate a list of hosts to give to prometheus for stats generation"""
+import json
+import os
+import tempfile
+from sys import argv
+
+from ocflib.infra.hosts import hosts_by_filter
+
+hosts = []
+for vals in hosts_by_filter('(|(type=server)(type=desktop)(type=staffvm))'):
+    cn, ip, host_type = vals['cn'][0], vals['ipHostNumber'][0], vals['type'][0]
+
+    # Don't include hozer machines
+    if cn.startswith('hozer-'):
```

Comment:
Why not include hozers?

Reply:
I took this from `gen-munin-nodes`, but I guess it doesn't hurt to monitor hozers.

Reply:
I think including hozers would be fine, might be nice to see stats on them if some service is being tested, etc.